### PR TITLE
rewrite ansible tasks using YAML style

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

Jenkins Build# 7972

### CODE REVIEW

File: playbooks/roles/xserver/tasks/deploy.yml [hunk 1]
```diff
@@ -1,106 +1,140 @@
...
-  template: >
-    src=git_ssh.sh.j2 dest=/tmp/git_ssh.sh
-    owner={{ xserver_user }} mode=750
-  notify: restart xserver
+- name: Upload ssh script
+  template:
+    src: git_ssh.sh.j2
+    dest: "/tmp/git_ssh.sh"
+    owner: "{{ xserver_user }}"
+    mode: "750"
```
Comment (e0d) [inline playbooks/roles/xserver/tasks/deploy.yml]: make 0750

File: playbooks/roles/xserver/tasks/main.yml [hunk 2]
```diff
@@ -2,37 +2,39 @@
 # As of right now this role requires
 # access to the edX 6.00x repo which is not public
 ---
-
-- name: checking for grader info
-  fail: msg="You must define XSERVER_GRADER_DIR"
+- name: Checking for grader info
+  fail:
+    msg: "You must define XSERVER_GRADER_DIR"
   when: XSERVER_GRADER_DIR|length == 0
```
Comment (e0d) [inline playbooks/roles/xserver/tasks/main.yml]: I think that these checks are no longer necessary, can you confirm that undefined vars will result in errors?

**Comment by e0d** [comment]
:+1:  @edx/devops second review?

File: playbooks/roles/xserver/tasks/main.yml (hunk 2 above)
Comment (arbabnazar) [inline playbooks/roles/xserver/tasks/main.yml]: @e0d yes it is giving an error if these variable(s) is/are not defined. Thanks

**Pull request merged**